src/borg/crypto/low_level.pyx

```diff
@@ -67,7 +67,6 @@ cdef extern from "openssl/evp.h":
         pass
 
     const EVP_CIPHER *EVP_aes_256_ctr()
-    const EVP_CIPHER *EVP_aes_256_gcm()
```

Comment:
i intentionally did not remove gcm although some people have bad feelings about it. other people have also bad feelings about patents (even with the patent license for FOSS and openssl), so there would be gcm as option for them.

technically, I'ld also choose ocb over gcm.